adyenofficial.php

```diff
@@ -1099,8 +1066,10 @@ public function hookPaymentOptions()
 
                 $oneClickOption = new PrestaShop\PrestaShop\Core\Payment\PaymentOption();
                 $oneClickOption->setCallToActionText(
-                    $this->l(
-                        'Pay by saved ' . $storedPaymentMethod['name'] . " ending: " . $storedPaymentMethod['lastFour']
+                    sprintf(
+                        $this->l('Pay by saved %s ending: %s'),
```

Comment:
in case the translation adds one more %s then the function will show a php warning - I think it's better to break this translation up to 2 translations and then concatenating the text later